addon/components/search-dropdown/template.hbs

```diff
@@ -1,16 +1,15 @@
-<div class="osf-search" data-bind="fadeVisible: showSearch" style="display: none">
+<div class="osf-search">
     <div class="container">
         <div class="row">
             <div class="col-md-12">
-                <form class="input-group" data-bind="submit: submit">
-                    <input id="searchPageFullBar" name="search-placeholder" type="text" class="osf-search-input form-control" placeholder="Search" data-bind="value: query, hasFocus: true">
+                <form class="input-group" {{action "search" on="submit"}}>
+                    <input id="searchPageFullBar" onchange={{action "updateQuery"}} name="search-placeholder" type="text" class="osf-search-input form-control" placeholder="Search">
```

Comment:
I must be misunderstanding what you want here. 

I did this:
```         
 <input id="searchPageFullBar" value=target.value onchange={{action "updateQuery"}} name="search-placeholder" type="text" class="osf-search-input form-control" placeholder="Search">
````
which puts placeholder as target.value.  How do I reference this value to pass it into updateQuery?